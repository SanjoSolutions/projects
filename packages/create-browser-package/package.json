{
  "name": "@sanjo/create-browser-package",
  "version": "0.1.0",
  "type": "module",
  "description": "Create browser package",
  "keywords": [],
  "homepage": "https://github.com/SanjoSolutions/unnamed/tree/main/packages/create-browser-package",
  "license": "Unlicense",
  "types": "./index.d.js",
  "bin": "./bin/create-browser-package.js",
  "repository": {
    "type": "git",
    "url": "https://github.com/SanjoSolutions/unnamed.git",
    "directory": "packages/create-browser-package"
  },
  "dependencies": {
    "@sanjo/create-package": "^0.2.0",
    "@sanjo/read-json": "^1.0.0",
    "@sanjo/write-json": "^1.0.0",
    "ncp": "^2.0.0"
  },
  "devDependencies": {
    "@sanjo/diff-folders": "^1.0.0",
    "@sanjo/exec": "^1.0.0",
    "@sanjo/fs": "^1.0.0",
    "@sanjo/make-temporary-directory": "^1.0.0",
    "@sanjo/test-create-package": "^0.2.0",
    "@types/ncp": "^2.0.4"
  },
  "publishConfig": {
    "access": "public"
  }
}
