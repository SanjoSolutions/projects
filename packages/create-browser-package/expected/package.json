{
  "name": "@sanjo/test-package-for-create-browser-package",
  "version": "0.0.0",
  "type": "module",
  "description": "Description of test package",
  "keywords": [],
  "homepage": "https://github.com/SanjoSolutions/unnamed/tree/main/packages/test-package-for-create-browser-package",
  "license": "Unlicense",
  "main": "./index.js",
  "repository": {
    "type": "git",
    "url": "https://github.com/SanjoSolutions/unnamed.git",
    "directory": "packages/test-package-for-create-browser-package"
  },
  "scripts": {
    "build": "webpack --config webpack.prod.js",
    "build:watch": "webpack-dev-server --open --config webpack.dev.js"
  },
  "dependencies": {},
  "devDependencies": {
    "@sanjo/webpack": "^1.1.0"
  },
  "types": "./index.d.ts",
  "publishConfig": {
    "access": "public"
  }
}
